handlers/deck_handler.py

```diff
@@ -0,0 +1,74 @@
+from hearthstone import deckstrings
+import binascii
+import requests
+import json
+
+class DeckHandler():
+
+    def __init__(self, deck_response):
+        self.deck_response = deck_response
+        self.decks = self.get_decks()
+
+    def handle(self, input):
+        # Handling the message being just the deck string
+        if ' ' not in input:

```

Comment:
This would be handled by the "multiple words" check as well.` "foo".split() => ["foo"]`